### ISSUE TITLE
Allow the back button in studies to be focusable

### ISSUE THREAD

**Issue by clocksmith** [flutter/gallery#65]
Right now if you are inside of a study you cannot navigate to the back button with focus, as it is outside of the current navigator.

This issue is blocked by: flutter/flutter#47344

original link: https://github.com/material-components/material-components-flutter-gallery/issues/566
